recipes/orange3-geo/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "Orange3-Geo" %}
+{% set version = "0.1.6" %}
+{% set sha256 = "e36f910157982a8d6cb595ef569f925e6d824240a2e3f11c632084e51c077f4e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - orange3
+    - scikit-learn
+    - pandas
+    - scipy >=0.17
+    - shapely
+    - simplejson
+
+test:
+  imports:
+    - orangecontrib.geo
+
+about:
+  home: http://github.com/biolab/orange3-geo
+  license: GPL-3.0
+  license_family: GPL
```

Comment:
Please ask upstream to add a license file and include it in the `MANIFEST.in` to allow packaging it in the future. Also please add a comment with `license_file: <license filename>` where `<license filename>` is replaced with the license file name as added in the repo and potentially location if it is not in the top directory of the repo.